core/controllers/topic_editor.py

```diff
@@ -224,6 +227,37 @@ def delete(self, topic_id):
         topic_services.delete_topic(self.user_id, topic_id)
 
 
+class TopicRightsHandler(base.BaseHandler):
+    """A handler for returning topic rights."""
+
+    @acl_decorators.can_manage_rights_for_topic
```

Comment:
I don't entirely understand the point of this handler. Is it for viewing rights or for managing rights? I don't see any managing going on here...

Reply:
It's just for viewing rights, or, shall I merge TopicManagerRightsHandler and this? I didn't do this because the url for the former is /assign_manager. Shall I change the url to /manage_topic_rights and merge them?

Reply:
Also, as all those who can edit the topic should be able to get the rights object, changed the rule to can_edit_topic.